src/componentes/CampoSenha/index.jsx

```diff
@@ -0,0 +1,28 @@
+import './CampoSenha.css'
+import React from 'react'
+
+const CampoSenha = (props) => {
```

Comment:
Não acho que tem nada errado com essa forma aqui, mas principalmente quando a gente não usa Typescript, fica complicado de saber oq tá vindo como props. 
Um padrão que ajuda nisso é desestruturar com os nomes das propriedades esperadas. Deixo uma sugestão abaixo como ilustração.
Seria bom a gente reunir depois a galera toda do front pra definir alguns desses padrões que vamos utilizar.

Um artigo sobre o tema: https://medium.com/@lcriswell/destructuring-props-in-react-b1c295005ce0

```suggestion
const CampoSenha = ({aoAlterar}) => {
```